api-client/src/main/java/com/xing/api/resources/MessagesResource.java

```diff
@@ -0,0 +1,134 @@
+     * requested user.
+     * <p>
+     * Possible optional <i>query</i> parameters:
+     * <table>
+     * <tr>
+     * <th>Parameter Name</th>
+     * <th><b>Description</b></th>
+     * </tr>
+     * <tr>
+     * <td><b>limit</b></td>
+     * <td>Restrict the number of attachments to be returned. Must be a positive number. Default: 10</td>
+     * </tr>
+     * <tr>
+     * <td><b>offset</b></td>
+     * <td>Offset. Must be a positive number. Default: 0</td>
+     * </tr>
+     * <tr>
+     * <td><b>strip_html</b></td>
+     * <td>Specifies whether the message content should be stripped of HTML (true) or not (false). The default value is
+     * false.</td>
+     * </tr>
+     * <tr>
+     * <td><b>user_fields</b></td>
+     * <td>List of user attributes to return. If this parameter is not used, only the ID will be returned.</td>
+     * </tr>
+     * <tr>
+     * <td><b>with_latest_messages</b></td>
+     * <td>Number of latest messages to include. Must be non-negative. Default: 0, Maximum: 100</td>
+     * </tr>
+     * </table>
+     *
+     * @param userId Id of the {@linkplain com.xing.api.data.profile.XingUser user} to request its
+     * {@linkplain Conversation conversations}.
+     * @return A {@linkplain CallSpec callSpec object} ready to execute the request.
+     *
+     * @see <a href="https://dev.xing.com/docs/get/users/:user_id/conversations">'List user conversations' resource
+     * page</a>
+     */
+    public CallSpec<List<Conversation>, HttpError> getConversationsByUserId(String userId) {
+        return Resource.<List<Conversation>, HttpError>newGetSpec(api, "/v1/users/{id}/conversations")
+              .pathParam("id", userId)
+              .responseAs(list(Conversation.class, "conversations", "items"))
+              .build();
+    }
+
+    /**
+     * Starts a {@linkplain Conversation conversation} by sending the passed message to the recipients.
+     * The subject of a conversation cannot be changed afterwards.
+     * Basic members are not allowed to send messages to non-contacts.
+     * Premium members are limited to 20 messages to non-contacts per month.
+     *
+     * @param content Message text with max size of 16384 UTF-8 characters
+     * @param recipientIds Comma-separated list of recipients. There must be between one and 10 recipients.
+     * Sender cannot be included.
+     * @param subject Subject for conversation. Max. size is 255 UTF-8 characters
+     * @param userId Id of the {@linkplain com.xing.api.data.profile.XingUser user} starting the conversation
+     * @return A {@linkplain CallSpec callSpec object} ready to execute the request.
+     *
+     * @see <a href="https://dev.xing.com/docs/post/users/:user_id/conversations">'Create a conversation' resource page</a>
+     */
+    public CallSpec<Conversation, HttpError> createConversation(String content, String recipientIds, String subject,
```

Comment:
Ok, checked the docks, and it seams that a request with url `/v1/users/me/conversations` is not supported... just change the signature..